client/src/shared/header/header.html

```diff
@@ -6,4 +6,5 @@
         </a>
         <swt-translate></swt-translate>
     </div>
+    <swt-hamburger-menu></swt-hamburger-menu>
```

Comment:
Please, change the position of the the hamburger menu to absolute and make sure that on the resolution for smartphones the hamburger does not cover the language nav panel

Reply:
Fixed